TemplateHandler.csproj

```
@@ -2,10 +2,10 @@
 
   <PropertyGroup>
     <PackageType>Template</PackageType>
-    <PackageVersion>2.2.0</PackageVersion>
+    <PackageVersion>2.2.1</PackageVersion>
```

```suggestion
    <PackageVersion>2.3.0</PackageVersion>
```

Ich würde behaupten, dieser PR ist schon mehr als ein Bugfix :D

---

Hast Recht